handler.go

```diff
@@ -63,7 +62,18 @@ func StatusChecker(w http.ResponseWriter, r *http.Request) {
 		BasinStorageAddr: "0x4b1f4d8100e51afe644b189d77784dec225e0596",  // TODO: move to config
 	}
 
-	sc, err := storage.NewStatusChecker(ctx, cfg)
+	if err := r.ParseForm(); err != nil {
+		errMsg := fmt.Sprintf("failed to parse form: %v", err)
+		fmt.Println(errMsg)
+		http.Error(w, errMsg, http.StatusInternalServerError)
+		return
+	}
+	simulated := false
```

Comment:
adds a new flag `simulated`. it will skip the deduplication logic when set to true. so, at the time of development and testing we can create duplicate deals.